modules/ROOT/pages/lb-cert-upload.adoc

```diff
@@ -49,8 +49,8 @@ To add a certificate using Runtime Manager:
 image::lb-cert-upload-update.png[Add certificate option in the load balancer settings page]
 . Select *Choose File* to upload the public key and private key files, such as:
 
-** Public key file: `example-com-crt.pem`
-** Private key file: `example-com-private.pem`
+** Public key file (Combined Primary.crt & Intermediate_CA.crt into one file.  File should be converted to PEM extension.): `example-com-crt-public.pem`
```

Comment:
Let's abstain from using extensions or formats. Instead let's just say "concatenated file including the server certificate and the intermediate CA(s)'s certificate. Only PEM format is allowed."

Reply:
This sounds fine.